colors/dracula.vim

```diff
@@ -276,7 +285,7 @@ hi! link WarningMsg   DraculaOrangeInverse
 " Syntax: {{{
 
 " Required as some plugins will overwrite
-call s:h('MatchParen', s:fg, s:pink, [s:attrs.underline])
+call s:h('MatchParen', s:green, s:none, [s:attrs.underline])
```

Comment:
I like this new version in the screenshot, mostly because it makes it more clear which paren the cursor is on. I could go either way for the colors (green v. pink).

Reply:
I used the green for about a week and it grew on me. The pink didn't work as well for me because it wasn't _quite_ as vibrant enough. That's why I switched it.